File: WeLearnApp/src/main/java/com/welearn/WeLearnApp/dto/response/PageResponse.java
package com.welearn.WeLearnApp.dto.response;

import lombok.*;
import lombok.experimental.FieldDefaults;

import java.util.List;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
@FieldDefaults(level = AccessLevel.PRIVATE)
public class PageResponse <T> {
    long currentPage;
    long elementPerPage;
    long totalPage;
    long totalElement;
    List<T> data;
}

File: WeLearnApp/desktop-app/TutorApp/Models/ForAPI/Response/PageResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TutorApp.Models.ForAPI.Response
{
    public class PageResponse <T>
    {
        public int currentPage { get; set; }
        public int elementPerPage { get; set; }
        public int totalPage { get; set; }
        public int totalElement {  get; set; }
        public List<T> data { get; set; }
    }
}
